source/_integrations/supla.markdown

```diff
@@ -15,22 +15,25 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The [Supla](https://supla.org/) is an Open Source home automation system for ESP8266 based devices. It has its own set of protocols, its own firmware and commercially available devices (produced for example by [Zamel](https://zamel.pl/pl-PL/produkty/supla-sterowanie-wi-fi)).
+The Supla integration allows you to use [SUPLA](https://supla.org/) devices.
```

Comment:
We should adjust the integration name to match as well in this case. (Please note, this requires a PR in our Core codebase as well).

Reply:
Is renaming an integration "hard"? Or does HA have some way to keep track of the "old" name?

Reply:
Just change the name in the manifest file.

<https://github.com/home-assistant/core/blob/23fa500406e9d53836a128f028fa68709d076d11/homeassistant/components/supla/manifest.json>

That would be all :)

Reply:
From a quick look `SUPLA` rarely seems to be used standalone (`SUPLA-CLOUD`, `SUPLA-DEV`, `SUPLA-CLIENT`), except in the context of the site or the "ecosystem"/project as a whole...

>All you need to start your adventure with SUPLA

>What is SUPLA?
>    SUPLA is a constantly developing project based on an open software and open hardware enabling to build a simple building automation system allowing to operate appliances and other elements at your home or in the office.

So, in this case, it would look like it makes sense to go with `SUPLA` rather than `SUPLA-CLOUD` (or something more specific).

---

And I guess as part of the core PR, there's some strings (l10n/i18n) that probably want updating to match.

And the `title` changing in this file as part of this PR too.